src/ocean/io/stream/Data.d

```diff
@@ -450,7 +452,7 @@ class DataOutput : OutputFilter
         private final void eat (Const!(void)* src, size_t bytes)
         {
                 auto count = output.write (src[0..bytes]);
-                assert (count is bytes);
+                verify(count == bytes);
```

Comment:
Is there any difference in using `is` vs `==` for value types?

Reply:
For structs only (as they may provide own `opEqual`)